frontend/src/components/Table/ColumnSort.tsx

```diff
@@ -1,8 +1,9 @@
+import variables from '_variables.module.scss';
```

Comment:
No change required for this PR.  This looks like something that could be replaced by the styled theme variables now.